.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,20 @@
+name: SonarCloud Scan
+on:
+  push:
+    branches:
+      - next
```

Comment:
This means that it only runs on PRs targeting `next`, right? Seems reasonable 👍

Reply:
Yes, `next` is the main branch in sonarcloud for lime-elements: https://sonarcloud.io/project/branches_list?id=Lundalogik_lime-elements

I configured the pattern for long-lived branches to next and main. I suspect main should appear in the list above after the first scanned commit is merged to main, but I'm not entirely sure how it will work. Maybe we need to add main to the list here to issue another scan, maybe not.

Reply:
I'm not sure we need it to scan `main` too. So far, `main` has exactly the same commits as `next`, it's just trailing in time. Or, rather, it gets updated to the current state of `next` at semi-regular intervals. I have thought a bit about decoupling them more, but that's the current state anyway. So right now, any warnings on `main` would just be warnings we've already had on `next`, and we wouldn't fix them on `main` by any other means than fixing them on `next`, and then merging `next` to `main`.

Reply:
Oh, no, this isn't running on PRs targeting `next`, it's running on `next` once a PR has been merged.

That's perhaps a little late…? 🤔

It does limit the number of runs it has to do though, since if we run it on PRs, it will run every time a PR is updated with new code changes. Let's leave it like it is for now, and see if we feel any need to run it on PRs.